### PR TITLE
os vars to camelCase

### PR DESCRIPTION
**Pull request by saberlynx**
**Checklist**:

Changed variables in "OS" to camelCase. Pushing these changes one by one because they require testing.
- [] Tests
- [] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/gmp/models/os.js [hunk 1]
```diff
@@ -26,13 +26,13 @@ class OperatingSystem extends Asset {
     const ret = super.parseProperties(elem);
 
     if (ret.os) {
-      ret.average_severity = ret.os.average_severity
+      ret.averageSeverity = ret.os.average_severity
         ? parseSeverity(ret.os.average_severity.value)
         : undefined;
```
Comment (bjoernricks) [inline gsa/src/gmp/models/os.js]:
```suggestion
        : undefined;
    delete ret.average_severity;
```

File: gsa/src/gmp/models/os.js [hunk 2]
```diff
@@ -26,13 +26,13 @@ class OperatingSystem extends Asset {
...
-      ret.latest_severity = ret.os.latest_severity
+      ret.latestSeverity = ret.os.latest_severity
         ? parseSeverity(ret.os.latest_severity.value)
         : undefined;
```
Comment (bjoernricks) [inline gsa/src/gmp/models/os.js]:
```suggestion
        : undefined;
    delete ret.latest_severity;
```

File: gsa/src/gmp/models/os.js [hunk 3]
```diff
@@ -26,13 +26,13 @@ class OperatingSystem extends Asset {
...
-      ret.highest_severity = ret.os.highest_severity
+      ret.highestSeverity = ret.os.highest_severity
         ? parseSeverity(ret.os.highest_severity.value)
         : undefined;
```
Comment (bjoernricks) [inline gsa/src/gmp/models/os.js]:
```suggestion
        : undefined;
    delete ret.highest_severity;
```

**Comment by bjoernricks** [comment]
operatingsystems/filterdialog doesnt require a change. @saberlynx and me already discussed that. The underscore_names are still required for the backend.

**Comment by bjoernricks** [comment]
Same for table. The backend is still returning the underscore name.

**Comment by swaterkamp** [comment]
I see. It was a hasted request for changes, because I wasn't finished looking deeper into the things I mentioned, while the approval email came in. I just wanted to stop the merge, before I had seen the details ;)

**Pull request merged**